aspnetcore/migration/50-to-60/samples/WebRP31to60/Startup.cs

```diff
@@ -124,5 +124,40 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
     }
 }
 #endregion
+#elif V60E
+#region snippet_60e
+public class Startup
+{
+    public Startup(IConfiguration configuration)
+    {
+        Configuration = configuration;
+    }
+
+    public IConfiguration Configuration { get; }
+
+    public void ConfigureServices(IServiceCollection services)
+    {
+        services.AddRazorPages();
+    }
+
+    public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+    {
+        if (!env.IsDevelopment())
+        {
+            app.UseExceptionHandler("/Error");
+            app.UseHsts();
+        }
+
+        app.UseHttpsRedirection();
+        app.UseStaticFiles();
+        //app.UseRouting();
+
+        //app.UseEndpoints(endpoints =>
+        //{
+        //    endpoints.MapRazorPages();
+        //});
```

Comment:
I get that this is unnecessary when using `WebApplication` in Program.cs, but why is it commented out here? Isn't highlighting it enough to indicate that this is what's being removed in the following sample?